pkg/apis/controller/v1beta1/schedulingspec.go

```diff
@@ -48,6 +48,7 @@ type SchedulingSpec struct {
 type SchedulingSpecTemplate struct {
 	NodeSelector map[string]string `json:"nodeSelector,omitempty" protobuf:"bytes,1,rep,name=nodeSelector"`
 	MinAvailable int               `json:"minAvailable,omitempty" protobuf:"bytes,2,rep,name=minAvailable"`
+	RequeuingTimeMinutes int       `json:"requeuingTimeMinutes,omitempty" protobuf:"bytes,2,rep,name=requeuingTimeMinutes"`
```

Comment:
should we also add requeueCount int param?

Reply:
Done. Added more options too.